tests/www/views/conftest.py

```diff
@@ -140,79 +136,3 @@ def anonymous_client(app):
 @pytest.fixture
 def anonymous_client_as_admin(app):
     return client_without_login_as_admin(app)
-
```

Comment:
moved to "tests_common" - those are now used in both "airflow" and "fab provider" tests.